backend/open_webui/utils/auth.py

```
@@ -8,24 +8,26 @@
 import os
 
 
-from datetime import UTC, datetime, timedelta
+from datetime import datetime, timedelta
+import pytz
```

Ruff (0.12.2)

`pytz` imported but unused

Remove unused import: `pytz`

(F401)